docs/examples/autocomplete.njk

```diff
@@ -0,0 +1,46 @@
+---
+title: Autocomplete example
+layout: example.njk
+eleventyExcludeFromCollections: true
+---
+{% from "x-govuk/components/autocomplete/macro.njk" import xGovukAutocomplete %}
```

Comment:
Can we not import these components in the `example.njk` layout… oh remembering issue with Eleventy and macros… is this still an issue?

Reply:
Tried that. Didn’t work and couldn’t work out why. Any ideas?